def myfunc():
    raise Exception('This is broken')
